src/scripts/main.js

```diff
@@ -1,3 +1,61 @@
 'use strict';
 
-// write code here
+const append = document.querySelector('.append-row');
+const appendRemove = document.querySelector('.remove-row');
+const column = document.querySelector('.append-column');
+const columnRemove = document.querySelector('.remove-column');
+const table = document.querySelector('.field');
+const tbody = table.querySelector('tbody');
+
+// Додавання стовпця
+append.addEventListener('click', () => {
+  const rows = table.rows;
```

Comment:
Consider renaming the variable 'rows' to something more descriptive like 'tableRows' to improve code readability.